apigentools/commands/init.py

```diff
@@ -16,6 +16,52 @@
 
 
 class InitCommand(Command):
+    config_file_json = {
+                    "codegen_exec": "openapi-generator",
```

Comment:
These look over-indented. I believe these should be 4 spaces past the variable name.

Reply:
Ok, I un-indented them -- let me know if they look ok.